tests/test_tools.py

```diff
@@ -2093,17 +2091,17 @@ def test_filter_employment_for_advisees(inp, expected):
     {"statuses" : ["all"], "types" : ["poster"]},
     [{"_id": "ghi",
     "authors": "tony stark, natasha romanov",
+    "begin_date": "2019-01-02",
```

Comment:
this is good.  btw, I think that good behavior for the function would be that it passes back date even if it was passed day, month, year.     Please use `get_dates` if it is not already being used.